File: 2014-10-CSharp-Fundamentals-1/6. Loops/ForLoops/ForLoops.cs
using System;

class ForLoops
{
    static void Main()
    {
        for (int num = 0; num < 10; num++)
        {
            Console.Write(num + " ");
        }
        Console.WriteLine();
        Console.WriteLine();

        int n = 5;
        decimal factorial = 1;
        for (int i = 1; i <= n; i++)
        {
            factorial *= i;
        }
        Console.WriteLine("{0}! = {1}", n, factorial);
        Console.WriteLine();

        for (int i = 1, sum = 1; i <= 128; i = i * 2, sum += i)
        {
            Console.WriteLine("i={0}, sum={1}", i, sum);
        }
        Console.WriteLine();
    }
}

File: 2015-10-Java-Fundamentals-1/06. Loops/demos/ForLoops.java
import java.math.BigInteger;

public class ForLoops {
  public static void main(String[] args) {
    for (int i = 0; i < 10; ++i) {
      System.out.printf("%d ", i);
    }
    System.out.println();

    int number = 5;
    BigInteger factorial = BigInteger.ONE;
    for (int i = 1; i <= number; i++) {
      factorial = factorial.multiply(BigInteger.valueOf(i));
    }
    System.out.printf("%d! = %s%n", number, factorial);

    for (int i = 1, sum = 1; i <= 128; i = i * 2, sum += i) {
      System.out.printf("i = %d, sum = %d%n", i, sum);
    }
  }
}
